integration_test/third_party_apps_data/applications/mongodb/centos_rhel/install

```diff
@@ -1,25 +1,26 @@
 set -e
 
-cat << EOF > mongodb-org-5.0.repo
```

Comment:
IIRC MongoDB 5 isn't available on Debian 11 ARM, so for simplicity I just upgraded all the tests to 6.0. This also unlocks testing on RHEL 9 which had no coverage before.

I don't want to keep piling more scope onto this PR so I'd rather not start a `mongodb5.0` test set here. If MongoDB 5 testing on RHEL is still valuable then perhaps we can special-case it so that 6.0 gets installed on RHEL 9 but 5.0 gets installed on other distros, as an interim measure while we work on a separate PR to add a `mongodb5.0` test set? WDYT?

Reply:
I'll leave it to you and ObservIQ (@dehaansa?) to determine how important it actually is to have 5.0 in our testing matrix. FWIW, I wasn't suggesting special-casing RHEL 9. It should be fairly straightforward to copy the old `mongodb` directory into `mongodb5.0` and exclude both `debian-11-arm64` and `rocky-linux-9` via `platforms_to_skip`, but we could do that in a separate PR if we decide it's worth it.

Reply:
Are there significant breaking changes between 5.0 and 6.0? If not, I don't see a requirement to continue to test 5.0, we aren't attempting to cover the full scope of supported versions in integration tests for other integrations, just versions where we encountered significant issues or know of breaking changes between versions. That said, if we want to add it as another supported integration to maintain support we can add 5.0 in the described way.

Reply:
Good point. I don't see anything in the [documentation](https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/) that could be potentially related to the way we collect metrics or logs, so maybe switching to 6.0 is not as big of a deal as I initially thought. Let's hold off on adding 5.0-specific testing until we discover breakage that wasn't caught by these.

Reply:
SGTM 👍